website/docs/r/rds_cluster.html.markdown

```diff
@@ -229,7 +229,7 @@ This argument supports the following arguments:
   The database can't be deleted when this value is set to `true`.
   The default is `false`.
 * `enable_global_write_forwarding` - (Optional) Whether cluster should forward writes to an associated global cluster. Applied to secondary clusters to enable them to forward writes to an [`aws_rds_global_cluster`](/docs/providers/aws/r/rds_global_cluster.html)'s primary cluster. See the [Aurora Userguide documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-global-database-write-forwarding.html) for more information.
-* `enable_http_endpoint` - (Optional) Enable HTTP endpoint (data API). Only valid when `engine_mode` is set to `serverless`.
+* `enable_http_endpoint` - (Optional) Enable HTTP endpoint (data API). Only valid when `engine_mode` is set to `serverless` or (if engine is set to aurora-postgresql) to `provisioned`.
```

Comment:
While this suggestion is wordier, I think it untangles the sentence a bit.

```suggestion
* `enable_http_endpoint` - (Optional) Enable HTTP endpoint (data API). Only valid when `engine_mode` is set to `serverless` or when `engine` is set to `aurora-postgresql` and `engine_mode` is set to `provisioned`.
```

Reply:
Thanks for the suggestion! I backed out the change to the cdktf/python version and updated the wording as asked.